src/gulp-plugins/gulp-mdtoXSLT.js

```diff
@@ -42,7 +42,7 @@ module.exports = function mdtoXSLT(options) {
     return through.obj(function(file, enc, callback) {
         var context = getContext(file);
         var t = this;
-        output(context);
+        // output(context);
```

Comment:
Can you add the debug: false param back in to lib/mdToXslt{Blocks,Pages}.js rather than removing the code here? That should stop the debug info rendering with every update, but allows us to view it when necessary.